MC-33183: New In-store delivery feature. Updated existing information and added new topics. (#604)
magento/merchdocs: src/shipping/shipping-in-store-delivery.md

---
title: In-store Delivery
---

> The In-store delivery method allows to select a source to be used as a pickup location during the checkout of your order.

Reviewer: ```suggestion
With the in-store delivery method,  you can select a source to be used as a pickup location during the checkout.
```

Author: done!, and I agree with you Kevin, this is something we can raise with someone? (ENG or Product?), and then we can work on applying necessary changes?. Count me in!

![]({% link images/images/luma-in-store-example.png %}){: .zoom}

> [_Example of a In-store delivery method during checkout_]({% link sales/checkout-step1-shipping.md %})

Reviewer: ```suggestion
[_In-store Delivery Method at Checkout_]({% link sales/checkout-step1-shipping.md %})
```

Author: done!

> ## To set up In-store Delivery

Reviewer: ```suggestion
## Set up In-store Delivery
```

Author: done!

> The first step is to complete the default settings for In-store Delivery. You can complete this step without changing the scope of the configuration.

Reviewer 2: Rewrite this paragraph. 

Enable In-store Delivery and optionally change the default values.

Author: done

1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.

> 1. In the _Sales_ section in the left panel, choose **Delivery Methods**.

Reviewer: ```suggestion
1. In the left panel, expand **Sales** and choose **Delivery Methods**.
```

Author: reviewed!

1. Expand ![]({% link images/images/btn-expand.png %}) the **In-Store Delivery** section.

1. Set **Enabled** to `Yes`.

> 1. If necessary, clear the **Use system value** checkbox to edit for any field to be edited.

Reviewer: ```suggestion
   {:.bs-callout-info}
   If needed, clear the **Use system value** checkbox to change the default for any field.
```

Author: nice, thanks!

> 1. Enter the **Method Name** that you want to appear as a label next to the calculated rate in the shopping cart.

Reviewer 2: I provided more detail earlier in this PR. Consider adapting that text for this entire procedure.

Author: modified, thanks!

1. Enter the **Title** that you want to appear for In-Store Delivery section during checkout.

    {:.bs-callout-info}
    The default title is `In-Store Pickup Delivery`.

> 1. Enter the **Price** that you want to charge for In-Store Delivery shipping.

Reviewer 2: ```suggestion
1. Enter the **Price**  to charge customers for the in-store pickup service.
```

Author: done!

1. Enter the **Search Radius** in kilometers for store pickup location search on storefront checkout.

> 1. In the **Displayed Error Message** box, type the message that appears if In-Store Delivery becomes unavailable. ()

Reviewer: ```suggestion
1. For **Displayed Error Message**, enter the message that appears if In-Store Delivery becomes unavailable.
```

Author: done!

    {:.bs-callout-info}
    The default message is `In-Store Delivery is not available. To use this delivery method, please contact us.`

> 1. Click **Save Config**.

Reviewer: ```suggestion
1. Click <span class="btn">Save Config</span>.
```

Author: modified!, thanks!
